servers/zms/src/main/java/com/yahoo/athenz/zms/store/impl/jdbc/JDBCConnection.java

```diff
@@ -6713,6 +6845,31 @@ private void addTagsToGroups(Map<String, Group> groupMap, String domainName) {
         }
     }
 
+    Map<String, Map<String, TagValueList>> getDomainResourceTags(String domainName, String caller, String sqlStatement) {
```

Comment:
If I understand correctly - this function comes to replace the specific function per resource.
you can address this change as part of this pr or once it merged in different pr.

and, I think, `getResourceTags` is better naming

Reply:
do we want to make it more generic? we said that this file should be simple as possible